tests/regression/lp-1848567/task.yaml

```
@@ -0,0 +1,44 @@
+summary: ensure apparmor_parser is not using gobs of memory
+details: |
+  Ensure that apparmor_parser is not using excessive amounts of memory when
+  compiling snap-update-ns profile of an application that looks like
+  gnome-system-monitor snap.
+prepare: |
+  # Pack, install and connect the three participants.
+  snap pack test-snapd-app
+  snap pack test-snapd-gnome-3-28-1804
+  snap pack test-snapd-gtk-common-themes
+
+  snap install --dangerous test-snapd-app_1_all.snap
+  snap install --dangerous test-snapd-gnome-3-28-1804_1_all.snap
+  snap install --dangerous test-snapd-gtk-common-themes_1_all.snap
+
+  snap connect test-snapd-app:gnome-3-28-1804 test-snapd-gnome-3-28-1804
+  snap connect test-snapd-app:icon-themes test-snapd-gtk-common-themes:icon-themes
+  snap connect test-snapd-app:sound-themes test-snapd-gtk-common-themes:sound-themes
+execute: |
+  # TODO: expand this test to install GNU time or find a suitable replacement.
+  if snap debug sandbox-features --required apparmor:kernel:mount && [ "$(command -v time)" != time ]; then
+    # Re-compile the apparmor profile for snap-update-ns for the
+    # gnome-system-monitor snap while ensuring that the profile is not loaded
```

this snap isn't actually named `gnome-system-monitor`?

```suggestion
    # test-snapd-app snap while ensuring that the profile is not loaded
```

---

This is a leftover from when I tested the real snap. I since re-wrote that to be a shim of the real thing that won't drift over time. Thanks for catching this.